### ISSUE TITLE
Service view PDFs

### ISSUE THREAD

**Issue by albanobattistella** [Stirling-Tools/Stirling-PDF#1409]
Hi,
when you choose the service to view PDF and add words, the welcome page is not translated.

[Welcome-1.pdf](https://github.com/user-attachments/files/15743912/Welcome-1.pdf)

**Comment by Frooodle** [comment]
yeah since it uses a actual PDF file the text cant be translated :(
Not sure of a good way round this

I will give it some thought
